examples/resnet-runtime.cpp

```diff
@@ -137,17 +137,14 @@ int main(int argc, char **argv) {
   std::vector<size_t> inputShape{1, 3, 224, 224};
 
   Placeholder *input;
-  std::vector<Placeholder *> inputs;
-  std::vector<PlaceholderList> phLists;
-  for (unsigned int i = 0; i < numDevices; i++) {
-    std::unique_ptr<Module> module = llvm::make_unique<Module>();
-    TypeRef inputType = module->uniqueType(ElemKind::FloatTy, inputShape);
-    input = loadResnet50Model(inputType, module.get(), i);
-    inputs.push_back(input);
-    llvm::outs() << "Adding to HostManager\n";
-    phLists.push_back(module->getPlaceholders());
-    EXIT_ON_ERR(hostManager->addNetwork(std::move(module)));
-  }
+  PlaceholderList phList;
+
+  std::unique_ptr<Module> module = llvm::make_unique<Module>();
+  TypeRef inputType = module->uniqueType(ElemKind::FloatTy, inputShape);
+  input = loadResnet50Model(inputType, module.get(), 0);
+  llvm::outs() << "Adding to HostManager\n";
```

Comment:
Do we want to keep this in? Probably not

Reply:
I kinda like it, it's a nice progress indicator.

Reply:
We don’t just print to stdout whenever we like it. Glow is a library. You can use glog if you want to report progress.